src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclint/resources/doclint.properties

```diff
@@ -57,6 +57,7 @@ dc.no.alt.attr.for.image = no "alt" attribute for image
 dc.no.summary.or.caption.for.table=no summary or caption for table
 dc.param.name.not.found = @param name not found
 dc.ref.not.found = reference not found
+dc.return.not.first = '{@return'} not at beginning of description
```

Comment:
I can see an inconsistency here. While the templates for `@return` and `@code` put an apostrophe immediately before the closing `}`, the template for `@value` (further below in this file) puts an apostrophe immediately after the closing `}`.

Reply:
I'll investigate and follow up.  It looks like general confusion whether to escape the curly braces, or to quote the enclosed text in the generated message.

Reply:
It looks like the quotes are never needed. I'll remove all of them.

````
/**
 * This is a comment.
 * This is a bad {@return fred}
 * This is a bad {@value}
 * This is a bad <code>{@code code}</code>.
 */
public class C { }
````

````
play/doclint-msgs/src/C.java:3: warning: {@return} not at beginning of description
 * This is a bad {@return fred}
                 ^
play/doclint-msgs/src/C.java:3: error: invalid use of @return
 * This is a bad {@return fred}
                 ^
play/doclint-msgs/src/C.java:4: error: {@value} not allowed here
 * This is a bad {@value}
                 ^
play/doclint-msgs/src/C.java:5: warning: {@code} within <code>
 * This is a bad <code>{@code code}</code>.
                       ^
2 errors
2 warnings
````